flytekit/remote/remote_fs.py

```diff
@@ -143,8 +143,9 @@ async def _put_file(
         Make the request and upload, but then how do we get the s3 paths back to the user?
         """
         prefix = kwargs.pop(_PREFIX_KEY)
+        _, str_digest, _ = hash_file(pathlib.Path(lpath))
```

Comment:
Changing how files are named in `_put_file` feels very impactful. (I'm not sure of the consequences)

Do you think it'll be easier to update  https://github.com/flyteorg/flytekit/blob/55e9f8d7f226302277c5d70abdcfd5d1ff19a789/flytekit/types/pickle/pickle.py#L41-L50 and rename itself (with `os.rename`?

Reply:
Moved to to_pickle. I initially placed it here because I encountered the same error with other types, but I can’t reproduce the issue now. I’ll revisit it once I’m able to reproduce it.